package.json

```diff
@@ -39,6 +39,7 @@
     "@babel/plugin-transform-react-jsx": "7.10.1",
     "@babel/preset-env": "7.10.2",
     "@jest/core": "26.1.0",
+    "@kristoferbaxter/google-closure-compiler": "20200517.1.0",
```

Comment:
Is the plan to eventually move this to `@ampproject`?

Reply:
Yes, once we've stabilized the implementation I'd like to move this under the `ampproject`.

Another option would be to move the java and graal compilation steps into the `amphtml` repo. Either way makes sense to me, but would prefer you and the @ampproject/wg-infra group to decide.